src/validators/v1/default-rule.js

```diff
@@ -58,6 +58,10 @@ module.exports = {
 			.withMessage('The requester_roles field must have at least one role.')
 			.custom((roles) => roles.every((role) => typeof role === 'string'))
 			.withMessage('All elements in the requester_roles array must be strings.')
+			.custom((roles) => roles.every((role) => /^[a-z_]+$/.test(role)))
```

Comment:
@sumanvpacewisdom it allows only small letter, allow Caps also

Reply:
roles are created with only small letters, based on that i have put the validations @rakeshSgr